modules/economy.js

```diff
@@ -1,293 +1,235 @@
 
-    client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, member.id], (err, result) => {
-      if (err) {
-        client.logError(err)
-        client.log.error(err)
-      }
-      if (Object.prototype.hasOwnProperty.call(result, 0)) {
-        client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(result[0].amount) + plusNumber), member.id, guild.id], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      } else {
-        client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`, `amount`) VALUES (?, ?, ?)', [member.id, guild.id, plusNumber], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      }
+    module.exports.getMemberInventoryAndBalance(client, member, guild, (memberInventoryAndBalance) => {
+      client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(memberInventoryAndBalance.amount) + dailyMoney), member.id, guild.id], (err) => {
+        if (err) client.logError(err)
+        if (err) throw new Error('DB_ERROR')
+        if (callback && !err) callback(dailyMoney)
+      })
     })
-
-    if (callback) callback(plusNumber)
     EgM.finish()
   },
   getWorkMoney: async (client, member, guild, callback) => {
     const EgM = client.Sentry.startTransaction({
       op: 'economy.getWorkMoney',
       name: 'Economy (getWorkMoney)'
     })
-    const plusNumber = Math.floor(Math.random() * 1500) + 1000
+    const workMoney = Math.floor(Math.random() * 1500) + 1000
 
-    client.pool.query('SELECT * FROM `guildEconomyUserBank` WHERE guild = ? AND member = ?', [guild.id, member.id], (err, result) => {
-      if (err) {
-        client.logError(err)
-        client.log.error(err)
-      }
-      if (Object.prototype.hasOwnProperty.call(result, 0)) {
-        client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(result[0].amount) + plusNumber), member.id, guild.id], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      } else {
-        client.pool.query('INSERT INTO `guildEconomyUserBank` (`member`, `guild`, `amount`) VALUES (?, ?, ?)', [member.id, guild.id, plusNumber], (err) => {
-          if (err) {
-            client.logError(err)
-            client.log.error(err)
-          }
-        })
-      }
+    module.exports.getMemberInventoryAndBalance(client, member, guild, (memberInventoryAndBalance) => {
+      client.pool.query('UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?', [(parseInt(memberInventoryAndBalance.amount) + workMoney), member.id, guild.id], (err) => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/alexandrumuntenas/pingu/pullRequest?prid=8657799)
```suggestion
      client.pool.query("UPDATE `guildEconomyUserBank` SET `amount` = ? WHERE `member` = ? AND `guild` = ?", [(parseInt(memberInventoryAndBalance.amount) + workMoney), member.id, guild.id], (err) => {
```